src/common/lib/utils.ts

```diff
@@ -96,9 +97,15 @@ const utils = {
   openUrl: (url: string) => {
     browser.tabs.create({ url });
   },
+  /**
+   *
+   * @param message
+   * @metadata is optional, so that it doesnt conflict with lnurl openPrompt function call
```

Comment:
What's the benefit of the JDDoc comment if we use TS? I'm also confused because it looks like `openPrompt` is getting `message` and `metadata` but actually it's getting a message with optional metadata? Maybe I'm reading it wrong.

Reply:
Not Sure, We were already using JSDoc format so made a quick comment using the same format

Reply:
Though I documented it in the wrong way corrected this! thanks for pointing out. It shall be ```@param message.metadata .....```

Reply:
shall I remove the whole comment then?  which format I shall follow currently?

Reply:
I'd rather get rid of JSDoc comments as logn as we don't really do anything with it. @bumi ?

Reply:
done